### PR TITLE
4.4 rel 2

### PR DESCRIPTION
**Pull request by mramuta**

### CODE REVIEW

**Comment by Bontgoy** [comment]
If requesting feedback make sure to wait for my reply before merging the pull request

File: ruby/vampires.rb [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
 year_born = gets.chomp.to_i
 puts "Our company cafeteria serves garlic bread. Should we order some for you?"
-want_garlic_bread = gets.chomp
+want_garlic_bread = gets.chomp.downcase
 puts "Would you like to enroll in the company’s health insurance?"
-want_health_insurance = gets.chomp
+want_health_insurance = gets.chomp.downcase
+
+hates_garlic_bread = want_garlic_bread.include? "no"
+correct_age = (age == 2016-year_born || age == 2016-year_born-1)
```
Comment (Bontgoy) [inline ruby/vampires.rb]: with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.now.year`

File: ruby/vampires.rb [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+if  correct_age && !hates_garlic_bread
+	vampire_status = "Probably not a vampire."
+end
+if !correct_age && (waives_health_insurance || hates_garlic_bread)
+	vampire_status = "Probably a vampire."
+end
+if !correct_age && (waives_health_insurance && hates_garlic_bread)
+	vampire_status = "Almost certainly a vampire."
+end
+if (name.include? "drake cula") || (name.include? "tu fang")
```
Comment (Bontgoy) [inline ruby/vampires.rb]: I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.

```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```

**Comment by mramuta** [comment]
Thank you very much for the feedback, sorry I merged it before you
commented. I just edited the pull request after merging it to include your
name after
​ reading the canvas announcement this morning.​

​​

**Comment by Bontgoy** [comment]
thanks!

**Pull request merged**